upgrade/1.2/scripts/upgrade/prerequisites.sh

```diff
@@ -298,6 +298,58 @@ else
     echo "====> ${state_name} has been completed"
 fi
 
+state_name="UPGRADE_UNBOUND"
+state_recorded=$(is_state_recorded "${state_name}" $(hostname))
+if [[ $state_recorded == "0" && $(hostname) == "ncn-m001" ]]; then
+    echo "====> ${state_name} ..."
+    {
+    manifest_folder='/tmp'
+    dns_forwarder=$(kubectl -n loftsman get secret site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d|yq r - spec.network.netstaticips.system_to_site_lookups)
+    system_name=$(kubectl -n loftsman get secret site-init -o jsonpath='{.data.customizations\.yaml}' | base64 -d|yq r - spec.network.dns.external)
+    unbound_version=$(ls ${CSM_ARTI_DIR}/helm |grep cray-dns-unbound|sed -e 's/\.[^./]*$//'|cut -d '-' -f4)
+
+    echo "ERROR: null value found.  See list of variables"
+    echo "dns_forwarder is $dns_forwarder."
+    echo "system_name is $system_name."
+    echo "unbound_version is $unbound_version."
+
+    if [ -z "$dns_forwarder" ] || [ -z "$system_name" ] || [ -z "$unbound_version" ]; then
+      exit 1
+    fi
+
+    cat > $manifest_folder/unbound.yaml <<EOF
+apiVersion: manifests/v1beta1
+metadata:
+  name: unbound
+spec:
+  charts:
+  - name: cray-dns-unbound
+    namespace: services
+    source: csm
+    values:
+      domain_name: $system_name
+      forwardZones:
+      - forwardIps: [$dns_forwarder]
+        name: .
+      global:
+        appVersion: $unbound_version
+      localZones:
+      - localType: static
+        name: local
+    version: $unbound_version
+EOF
+
+echo "$manifest_folder/unbound.yaml"
+cat /tmp/unbound.yaml
```

Comment:
Minor, but if we're using $manifest_folder, then we should use it on line 343, too.